EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+
+#### Prerequisites
+
+- You have the message specification of your business partner at hand.
+- You have created the basic definition settings for this type of message and business partner, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab*
+
+![Exporter definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Exporter definitions]")
+
+1. Click the definition for which you want to specify the header file settings.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+2. Click the *File settings* tab.   
+    The *File settings* tab is displayed.
+
+     ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefDefineFileSettings.png "[File settings tab]")
+
+3. If you have not already done so, click the *Release* dropdown list and select the required release.
+
+4. If you have not already done so, click the *Message type* dropdown list and enter the required message type.
+
+5. Click the ![Get](../../Assets/Icons/Download.png "[Get]") [GET] button.   
+    The message structure is displayed below. The message name is highlighted in red because it is still empty.
+
+    ![Message](../../Assets/Screenshots/EDI/Operation/ExportDefMessNew.png "[Message]")
+
+6. Expand the message structure by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button.   
+    The message structure with its single segments is displayed. All segments that contain mandatory content in the EDIFACT standard, are indicated with an M[Number] in italic and bold, for example ***M35***.    
+    The ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button has changed to an ![Expanded](../../Assets/Icons/NodeExpanded.png "[Expanded]") (Expanded) button.
+
+    ![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructure.png "[Message structure]")
+
+7. Continue with [Specify message name &frasl; constant text value](#specify-document-name-⁄-constant-text-value) described below. Alternatively, click the [SAVE] button in the upper right corner to save your settings.
+   After saving, the *Exporter definitions* view is displayed.
+
+
+
+## Specify document name &frasl; constant text value
+
+Start specifying the header file segments by filling out the single elements of the BGM (Beginning of message) segment.   
+
+This procedure shows how to edit a constant text value, for example the transaction type *Commercial invoice* that specifies the type of the INVOIC message.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
```

Comment:
Ich finde, the prerequisite sollte "You have got the header file segments, see ..." sein.
Dass man die "message structure" ausklappen soll, um in die Ausgangssituation zu gelangen, steht im Pfad vor dem Screenshot.

Reply:
done